ophyd/v2/core.py

```diff
@@ -155,6 +155,12 @@ def name(self) -> str:
         """Return the name of the Device"""
         return self._name
 
+    @property
+    def children(self) -> Generator[Tuple[str, Device], None, None]:
```

Comment:
We decided that `children` was a good name, but I have since thought that a `property` returning a generator is not a very common pattern. Should we make it a method instead?

Reply:
@tacaswell thoughts?

Reply:
FWIW, I agree that a method is common practice.